### ISSUE TITLE
Automatic periodic data export

### ISSUE THREAD

**Issue by ajsteele** [tszheichoi/awesome-sensor-logger#49]
Is it possible to periodically export data automatically, eg export a zip of CSV files every day, for continuous data collection? It’s annoying to have to manually stop the recording and export every so often if you want the app to be running continuously.

Thanks for the otherwise great app!

**Comment by tszheichoi** [comment]
Makes sense. I do have some ideas about some sort of rolling export and rolling upload mode. The challenge is to do this reliably in the background over long periods of time. While this is on the feature roadmap, I don't anticipate this to be available anytime soon.
